system_core/conf.yaml.example

```diff
@@ -1,6 +1,9 @@
 init_config:
 
 instances:
-  # No configuration is needed for this check.
+  # No configuration is needed for this check. However, you can add tags.
   # A single instance needs to be defined with any value.
   - foo: bar
+  # tags:
```

Comment:
The host tags are submitted as metadata and get overwritten in our current setup. We need to glue the tags to the metrics to circumvent that. The Datadog customer support has more information on that. On the other hand, the nginx and jmx integrations provide this possibility as well so why not add it to these ones.